.github/workflows/nightly.yml

```diff
@@ -49,10 +49,7 @@ jobs:
       - name: 🕵️ Check for changes
         id: version
         run: |
-          # get latest commit sha
-          SHA=$(git rev-parse HEAD)
-          # get first 7 characters of sha
-          SHORT_SHA=${SHA::7}
+          SHORT_SHA=$(git rev-parse --short HEAD)
```

Comment:
Slightly easier way to do this stolen from our `version.js` script - results in a 9 char SHA instead of 7.